src/main/java/com/jcabi/aspects/aj/MethodCacher.java

```diff
@@ -436,6 +458,7 @@ public boolean equals(final Object obj) {
          * Send a result through, with necessary logging.
          * @param result The result to send through
          * @return The same result/object
+         * @checkstyle DesignForExtensionCheck (2 lines)
```

Comment:
@nesterov-n now I'm thinking whether we shouldn't make this method abstract and implement it in concrete and fake object, but let's leave it to Yegor, for me it's fine for now